website/docs/foundations/breakpoints/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,42 @@
+## Breakpoints
+
+Breakpoints are the foundational building blocks for responsive design and define the viewport widths where the page layout adjusts and/or changes for varied screen sizes and devices.
+
+These values are then used to create utilities and helper components to standardize changes of the UI at different viewport sizes.
+
+### The ranges
+
+Helios defines five breakpoints: `sm`, `md`, `lg`, `xl`, and `xxl`.
+
+| Name   | Value        | Application                   |
+| ------ | ------------ | ----------------------------- |
+| sm     | 480px        | 480 → 767                     |
+| md     | 768px        | 768 → 1087                    |
+| lg     | 1088px       | 1088 → 1439                   |
+| xl     | 1440px       | 1440 → 1919                   |
+| xxl    | 1920px       | 1920 and up                   |
+
+The breakpoint name refers to a range of pixel values, not just the starting value. For example, the `sm` breakpoint refers to widths between 480px and 767px. This helps simplify the language around what these ranges mean when designing, and increases the flexibility in implementation.
+
+### Working with custom values
+
+The provided breakpoints serve as a cohesive _starting_ point for design. However, custom values may be necessary for specific use cases that these breakpoints do not cover. For example, if a team discovers that users with 2500px viewports are using a product, a nuanced approach to the UI at that screen size may be required. In such cases, an additional value can be added to enhance the user experience.
+
+!!! info
+
+If a custom value already exists that is relatively close to an established breakpoint, it is recommended that it be migrated to the nearest standard breakpoint. This creates a consistent and predictable experience for the user as they work within (and across) the HashiCorp product suite.
+
+!!!
+
```

Comment:
```suggestion
```

Reply:
Nit: Delete extra blank line (so that there's only one blank line vs. 2 in a row).